source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk1.rst

```diff
@@ -0,0 +1,183 @@
+^^^^^^^^^^^^^^^^^^
+
+The transport controls work mostly as labelled on the controller:
+
+:hwlabel:`SYNC` button
+    Press to :term:`sync` :term:`tempo`.
+    Press and hold to enable sync lock.
+    Press again to disable sync lock.
+    Press with :hwlabel:`SHIFT` to enable sync lock without needing to hold.
+:hwlabel:`CUE` button
+    Behavior depends on the :ref:`cue mode set in the Mixxx preferences <interface-cue-modes>`.
+    Press with :hwlabel:`SHIFT` to seek the beginning of the track and stop.
+:hwlabel:`PLAY` button
+    Play or pause the deck. Press with :hwlabel:`SHIFT` to toggle key lock.
+:hwlabel:`TEMPO` fader
+    Adjusts the :term:`tempo`.
+:hwlabel:`RESET` button
+    Mixxx `does not yet have a very useful flux/slip
+    mode <https://bugs.launchpad.net/mixxx/+bug/1475303>`__, so instead this
+    button toggles the top pad row to the intro/outro cues. Press the button
+    when it is lit to return the top pad row to hotcue mode.
+
+    When slip mode is implemented in Mixxx in the future, this could be
+    changed so :hwlabel:`SHIFT` + :hwlabel:`RESET` button activates intro & outro mode for the
+    pads.
+
+Mixer
+~~~~~
+
+Deck Columns
+^^^^^^^^^^^^
+
+:hwlabel:`GAIN` encoder
+    Controls the Quick Effect superknob for the deck.
+    With :hwlabel:`SHIFT`, controls gain.
+    Press to reset the Quick Effect superknob.
+    Press with :hwlabel:`SHIFT` to reset gain.
+:hwlabel:`FX 1`/:hwlabel:`FX 2` buttons
+    Assign the deck to effect units 1 and 2.
+:hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs
+    Adjust the high, middle, and low frequencies.
+:hwlabel:`CUE` button
+    Toggle whether the deck is routed to the :term:`prefader headphone output <PFL>`.
+    With :hwlabel:`SHIFT`, toggle quantize for the deck.
+Fader
+    Control the deck volume.
+
+Center Column
+^^^^^^^^^^^^^
+
+:hwlabel:`MAIN LEVEL` knob
+    Adjust the volume of the main output.
+    This acts on the controller’s audio interface output in hardware,
+    so it is not mapped to the main mix gain knob in Mixxx (otherwise the gain would be applied twice).
+:hwlabel:`SAMPLES` knob
+    Adjusts the gain of samplers 1-8.
+:hwlabel:`SAMPLES ON A`/:hwlabel:`SAMPLES ON B` buttons
+    Toggles the top pad row of the corresponding deck to control samplers.
+    Press when lit to return the pads to controlling :term:`hotcues <hotcue>`.
+:hwlabel:`BROWSE` encoder
+    Scroll through the music library.
+    Push to maximize the library browser on screen.
+    Turn with :hwlabel:`SHIFT` to scroll 5 rows at a time.
+:hwlabel:`LOAD A`/:hwlabel:`LOAD B` buttons
+    Load the track selected in the library to the corresponding deck.
+    Press with :hwlabel:`SHIFT` to unload a track.
+:hwlabel:`LEVEL` meters
+    The meters show the levels for each deck.
+Crossfader
+    Crossfade between the decks.
+
+Effects
+~~~~~~~
+
+The Kontrol S2 MK1 uses the `standard Mixxx effects
+mapping <https://github.com/mixxxdj/mixxx/wiki/standard-effects-mapping>`__.
```

Comment:
The manual should be self contained and should not link into the Wiki. Please copy what applies for this mapping.

Reply:
It is the same in the Mk2 manual

Reply:
I made it self contained instead is that fine to?